src/garage/experiment/local_runner.py

```diff
@@ -234,7 +243,10 @@ def train(self,
             pause_for_plot(bool): Pause for plot.
 
         Returns:
-            The average return in last epoch cycle.
+            float: The average return in last epoch cycle.
+
+        Raises:
+            Exception: if self.has_setup is False.
```

Comment:
Please create a new exception class for this. You can call it LocalRunnerException.

```python
class LocalRunnerException(Exception):
    pass
```